### PR TITLE
Fixes #683: Split users into users and presence

### LINKED ISSUE #683: Split users and presence in Redux state
Split users and presence in Redux state for performance reasons.
Message list depends on 'users' but not on 'presence'.
Currently, presence updates cause message list updates.

### PR DESCRIPTION
**Pull request by kunall17**
@borisyankov The presence is added when a user logs in (rather when the server sends this event type), and the whole user list is not added, when a person comes online the email is added to the presence state, Hence this adds new users as well which conflicts your previous way of doing this (That ignored the users) 

The timestamp and the status is saved and the online status can be determined, currently the presence is not used anywhere (though it is passed in ConversationCard and one more component)

Needs more testing, though!

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@kunall17, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by kunall17** [comment]
Works well, up for a review!

**Comment by borisyankov** [comment]
TL; DR; LGTM; 🥇
